### PR TITLE
`lang/go`: Add simple HTTP program

### PR DESCRIPTION
**Pull request by razvand**
Add source code, Makefile, prebuilt ELF, pre-etracted dynamic libraries. Document steps to (re)build and run in `README.md`.

### CODE REVIEW

File: lang/go/http_server/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Simple Go HTP Server
```
Comment (StefanJum) [inline lang/go/http_server/README.md:1]:
```suggestion
# Simple Go HTTP Server
```

**Review by StefanJum — CHANGES_REQUESTED** [review]
Works fine, same as the other prs, I would move anything else under `lang/go/` in specific directories.

**Comment by razvand** [comment]
@StefanJum , see now.

**Review by StefanJum — APPROVED** [review]
All good.
Reviewed-by: Stefan Jumarea <stefanjumarea02@gmail.com>
Approved-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Pull request merged**